CMakeLists.txt

```diff
@@ -1140,7 +1135,8 @@ endif()
 
 if(LINUX)
     message(STATUS "Building on Linux.")
-    add_definitions(-DLINUX -DPIPES -DNO_FLTK_THREADS -D_GNU_SOURCE -DHAVE_SOCKETS)
```

Comment:
`NO_FLTK_THREADS` was unused

Reply:
Some of these are probably irrelevant after we moved out the plugins to the `csound/plugins` repo. It's good to clear that up.